runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslatorTest.java

```diff
@@ -440,6 +441,7 @@ public void testDiskSizeGbConfig() throws IOException {
   }
 
   @Test
+  @Ignore("TODO: Reenable when the hierarchy is copied")
```

Comment:
For each TODO: I think some are fixed, some should be just deleted, and the rest should have JIRAs if possible.